### DISCUSSION TITLE
Game presentation

### DISCUSSION THREAD

**Discussion by beatrix29** [mamedev/discussions#119] [Q&A]
To whom may I concern as to an admin, editor, owner or similar? I want to discuss opportunities about providing an interesting game to the community and earnings to the website. Thanks in advance! :)

**Comment by happppp** [comment]
If you were being vague by accident and actually wanted to contribute something that MAME can emulate, please refer to: https://wiki.mamedev.org/index.php?title=Submitting_Source_Code#Submitting_ROM_or_disk_dumps

If it's about advertising: No.
